dashboard/src/components/PageHeader/PageHeader.scss

```diff
@@ -85,3 +87,14 @@
     color: var(--cds-global-typography-color-400, #333);
   }
 }
+
+header .search-box::before,
+header .search::before,
+.header .search-box::before,
+.header .search::before {
+  background-color: transparent;
+}
+
+header .search-box cds-icon {
+  color: var(--cds-global-typography-color-400, #333);
+}
```

Comment:
Fixing some colors in the dark/light mode:

![1](https://github.com/vmware-tanzu/kubeapps/assets/11535726/99b70970-6e75-4e34-ab3e-39014d89941a)